src/com/google/javascript/jscomp/Normalize.java

```diff
@@ -73,6 +73,12 @@
         "The use of scope variable {0} is not allowed within a catch block " +
         "with a catch exception of the same name.");
 
```

Comment:
1) Warnings should happen in checks; normalize is run too late. Move the warning to IRFactory.java.
2) Warn for every use of <code>@expose</code>, not just the first one.